experiments/long_runs/snowy_land_pmodel.jl

```diff
@@ -106,6 +108,21 @@ function setup_model(FT, start_date, stop_date, Δt, domain, toml_dict)
     soil_moisture_stress =
         ClimaLand.Canopy.PiecewiseMoistureStressModel{FT}(domain, toml_dict)
 
+    # Read spatially-varying canopy height from CLM data
+    raw_canopy_height = clm_canopy_height(surface_space)
+    # Cap height to stay below atmospheric reference height (ERA5 at 10m)
+    # Use 8m max to leave 2m buffer for atmospheric coupling
+    canopy_height =
+        effective_canopy_height(raw_canopy_height, FT(10.0); buffer = FT(2.0))
+
+    # Create biomass model with spatially-varying height
+    biomass = PrescribedBiomassModel{FT}(
```

Comment:
I think it would be good to add lines 112-116 into the PrescribedBiomassModel constructor, so it is not something that has to be updated each time (if we decide to include this as the default)